cob_hardware_config/common/cob3.rviz

```diff
@@ -205,64 +204,21 @@ Visualization Manager:
           Show Axes: false
           Show Trail: false
           Value: true
-        sdh_finger_11_link:
+        pg70_finger_left_link:
```

Comment:
will this be a problem with SDH equipped robots? I think not because the Robot model will be updated with the current robot_description parameter